backend/src/test/java/com/pickpick/service/ChannelSubscriptionServiceTest.java

```diff
@@ -30,43 +35,152 @@ class ChannelSubscriptionServiceTest {
     @Autowired
     private MemberRepository members;
 
-    @DisplayName("채널 구독을 저장")
+    @DisplayName("채널 구독을 단건 저장")
     @Test
-    void saveAll() {
+    void save() {
         // given
         Member member = saveMember();
-        List<Channel> findChannels = channels.findAll();
+        Channel channel = saveChannel("slackId", "채널 이름");
+        subscribeChannel(member, channel);
+
+        // then
+        assertThat(channelSubscriptionService.findAllOrderByViewOrder(member.getId())).hasSize(1);
+    }
+
+    @DisplayName("존재하지 않는 채널 ID로 채널 저장 시 에러 발생")
```

Comment:
```suggestion
    @DisplayName("존재하지 않는 채널 ID로 채널 구독 저장 시 에러 발생")
```
채널 저장이라고 하니까 구독이 아닌 새로운 채널을 생성한다고 오해할 수 있을 것 같아요!